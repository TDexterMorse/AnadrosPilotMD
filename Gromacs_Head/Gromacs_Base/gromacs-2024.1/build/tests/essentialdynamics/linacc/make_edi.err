                     :-) GROMACS - gmx make_edi, 2024.1 (-:

Executable:   /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/bin/gmx
Data prefix:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1 (source tree)
Working dir:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/essentialdynamics/linacc
Command line:
  gmx make_edi -f /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/essentialdynamics/linacc/eigenvec.trr -outfrq 2 -linacc 1 -accdir +1 -o sam.edi

trr version: GMX_trn_file (single precision)
Read non mass weighted reference structure with 4 atoms from /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/essentialdynamics/linacc/eigenvec.trr
Read non mass weighted average/minimum structure with 4 atoms from /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/essentialdynamics/linacc/eigenvec.trr
Read 3 eigenvectors (for 4 atoms)

Reading file topol.tpr, VERSION 2024.1 (single precision)
Reading file topol.tpr, VERSION 2024.1 (single precision)
Group     0 (         System) has     4 elements
Group     1 (          Other) has     4 elements
Group     2 (            DRG) has     4 elements
Select a group: 
Thanx for Using GROMACS - Have a Nice Day

