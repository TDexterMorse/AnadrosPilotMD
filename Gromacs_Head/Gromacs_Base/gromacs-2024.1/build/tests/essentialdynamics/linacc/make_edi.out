Eigenvector list -linfix consists of the indices: 
Eigenvector list -linacc consists of the indices: 1 
Eigenvector list  -flood consists of the indices: 
Eigenvector list -radfix consists of the indices: 
Eigenvector list -radacc consists of the indices: 
Eigenvector list -radcon consists of the indices: 
Eigenvector list    -mon consists of the indices: 

Select an index group of 4 elements that corresponds to the eigenvectors
Selected 0: 'System'

